### PR TITLE
feat: AFD Recommendation added

### PR DESCRIPTION
**Pull request by jimays-avila**
# Overview/Summary

Added a recommendation around avoiding using AFD classic to the recommendations list and added the Guid.KQL

## Related Issues/Work Items
AB#35190

https://dev.azure.com/CSUSolEng/Azure%20Infra%20Networking/_workitems/edit/35190/

Breaking Changes
N/A

## As part of this pull request I have

- [ ] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library-v2/contributing) and ensured this PR is compliant with the guide
- [ ] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/pulls)
- [ ] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/issues) or ADO Work Items (Internal Only)
- [ ] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/tree/main)
- [ ] Ensured PR tests are passing
- [ ] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries
- [ ] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

**Review by ejhenry — COMMENTED** [review]
@jimays-avila thanks for the contribution. Looks like four of the PR checks are failing. Let me know if you need help working through any of them.

**Comment by jimays-avila** [comment]
I will review it tomorrow and fix the issues @ejhenry thanks.

**Comment by jimays-avila** [comment]
@ejhenry I fixe most of the issue but am not sure how to fix the issue with for "Code Review - Linting & Link Checks / Lint code base".  Could you help me with this one? I am still looking into it in between my calls.

**Comment by oZakari** [comment]
Hey @jimays-avila, it looks like it's trailing white space. You can either remove the extra white space manually or I recommend installing https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig for VSCode which will look for configs in repos and then try to align to them so you don't have to worry about it. 
image

**Comment by jimays-avila** [comment]
@oZakari  thanks I looked over that and did not see that. Let me fix that now.

File: azure-resources/Cdn/profiles/recommendations.yaml [hunk 1]
```diff
@@ -296,3 +296,23 @@
   learnMoreLink:
     - name: Private link for Azure Front Door
       url: "https://learn.microsoft.com/azure/frontdoor/private-link"
+
+- description: Avoid using Classic AFD
```
Comment (oZakari) [inline azure-resources/Cdn/profiles/recommendations.yaml:300]:
```suggestion
- description: Avoid using Classic Azure Front Door
```

**Review by oZakari — APPROVED** [review]
LGTM

**Review by oZakari — APPROVED** [review]

File: azure-resources/Cdn/profiles/recommendations.yaml [hunk 2]
```diff
@@ -296,3 +296,23 @@
...
+
+- description: Avoid using Classic Azure Front Door
+  aprlGuid: 82fa3cff-74bd-4063-b726-834f160592fa
+  recommendationTypeId: null
+  recommendationControl: Service Upgrade and Retirement
+  recommendationImpact: High
+  recommendationResourceType: Microsoft.Cdn/profiles
+  recommendationMetadataState: Active
+  longDescription: |
+    Your Long Description Here
```
Comment (ejhenry) [inline azure-resources/Cdn/profiles/recommendations.yaml:308]:
```suggestion
```

**Review by ejhenry — APPROVED** [review]
LGTM

**Review by oZakari — APPROVED** [review]

**Pull request merged**